docs/sql-ref-syntax-qry-select-join.md

```diff
@@ -18,5 +18,52 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+### Description
 
-**This page is under construction**
+A SQL join is used to combine rows from two relations based on join criteria. The following section describes the overall join syntax and the sub-sections cover different types of joins along with examples.
+
+### Syntax
+{% highlight sql %}
+
+relation { [ join_type ] JOIN relation [ join_criteria ] | NATURAL join_type JOIN relation }
+
+{% endhighlight %}
+
+### Parameters
+<dl>
+  <dt><code><em>relation</em></code></dt>
+  <dd>
+    Specifies the relation to be joined.
+  </dd>
+  <dt><code><em>join_type</em></code></dt>
+  <dd>
+    Specifies the join type.<br><br>
+    <b>Syntax:</b><br>
+      <code>
+        [ INNER ]
+        | CROSS
+        | LEFT [ OUTER ]
+        | [ LEFT ] SEMI
+        | RIGHT [ OUTER ]
+        | FULL [ OUTER ]
+        | [ LEFT ] ANTI
+      </code>
+  </dd>
+  <dt><code><em>join_criteria</em></code></dt>
+  <dd>
+    Specifies how the rows from one relation will be combined with the rows of another relation.<br><br>
+    <b>Syntax:</b>
+      <code>
+      ON booleanExpression | USING ( column_name [ , column_name ... ] )
+      </code>
+  </dd>
+</dl>
+
+### Join Types
+- [INNER JOIN](sql-ref-syntax-qry-select-join-inner.html)
+- [LEFT JOIN](sql-ref-syntax-qry-select-join-left.html)
+- [RIGHT JOIN](sql-ref-syntax-qry-select-join-right.html)
+- [FULL JOIN](sql-ref-syntax-qry-select-join-full.html)
+- [CROSS JOIN](sql-ref-syntax-qry-select-join-cross.html)
+- [SEMI JOIN](sql-ref-syntax-qry-select-join-semi.html)
+- [ANTI JOIN](sql-ref-syntax-qry-select-join-anti.html)
```

Comment:
We need these individual pages for the join types? I feel most of contents (e.g., `employee` and `department`) are overlapped there...

Reply:
I will change to one page and see how it looks.

Reply:
Thanks!